freqtrade/pairlist/AgeFilter.py

```diff
@@ -26,12 +26,11 @@ def __init__(self, exchange, pairlistmanager,
         self._min_days_listed = pairlistconfig.get('min_days_listed', 10)
 
         if self._min_days_listed < 1:
-            raise OperationalException("AgeFilter requires min_days_listed must be >= 1")
+            raise OperationalException("AgeFilter requires min_days_listed be >= 1")
         if self._min_days_listed > exchange.ohlcv_candle_limit:
-            raise OperationalException("AgeFilter requires min_days_listed must not exceed "
+            raise OperationalException("AgeFilter requires min_days_listed be not exceeding "
```

Comment:
somehow this sounds odd (from a sentence / grammar perspecive).

maybe 
```suggestion
            raise OperationalException("AgeFilter requires min_days_listed to be below "
```

Reply:
In this case I definitely prefer the original wording.